frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -1207,7 +1206,7 @@ export const ClusterSettingsPage: React.FC = () => {
   };
   return (
     <PageTitleContext.Provider value={titleProviderValues}>
-      <PageHeading title={<div data-test-id="cluster-settings-page-heading">{title}</div>} />
+      <PageHeading title={<div data-test-id="cluster-settings-page-heading">{title}</div>} detail />
```

Comment:
Add the border above the tabs since it lacks them and the standard is for pages with tabs to have the border.

Reply:
To better match with PF I think that the PageHeading border should be removed here as well

Reply:
Oh.  I missed this one.  I think I agree.  Gonna discuss this one with the broader community.

Reply:
Thinking we should tackle this as a follow on so as not to conflate the issues.